SCF/SCF_Matt.dat

```diff
@@ -1,7 +1,8 @@
 #SCF Implementation with DIIS using Psi4 molecule, MintsHelper, and wfn.
+
 import numpy as num
```

Comment:
`num` as an abbreviation for `numpy` is very non-standard. I would either use the abbreviation `np` or the full `numpy`.